includes/wikia/resourceloader/ResourceLoaderGlobalWikiModule.class.php

```diff
@@ -84,6 +90,57 @@ protected function createTitle( $titleText, $options = array() ) {
 		return $title;
 	}
 
+	/**
+	 * Create title for scripts. Only NS_MEDIAWIKI is allowed for javascript pages.
+	 *
+	 * @param string $titleText
+	 * @param array $options
+	 * @return GlobalTitle|null|Title
+	 * @throws MWException
+	 */
+	private function createScriptTitle( $titleText, $options ) {
+		global $wgCityId;
+
+		$title = null;
+		$external = false;
+
+		list( $text, $namespace ) = $this->parseTitle( $titleText );
+
+		if ( !empty( $options['city_id'] ) && $wgCityId != $options['city_id'] && $text !== false ) {
```

Comment:
Since `$options['city_id']` is used a few times here, you could store it in a local variable and cast it to integer to avoid using soft `==` checks.